### PR TITLE
Test MySQL against Windows now

### PR DESCRIPTION
**Pull request by caseymcc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,32 +41,21 @@ matrix:
...
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505608
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=sanitize-leak
-
-    #  FIXME:
-    #  * https://travis-ci.org/ingenue/hunter/jobs/112505609
-    # - os: linux
-    #   env: PROJECT_DIR=examples/MySQL-client TOOLCHAIN=sanitize-thread
-
+    # FIXME: https://travis-ci.org/caseymcc/hunter/jobs/218630987
```
Comment (ruslo) [inline .travis.yml]: What changed here?

**Review by ruslo — CHANGES_REQUESTED** [review]

**Comment by ruslo** [comment]
Will test this first:
* https://github.com/ruslo/hunter/pull/755

Then if everything is okay (we don't break old stuff), we can add new toolchains.

File: .travis.yml (hunk 1 above)
Comment (caseymcc) [reply .travis.yml]: probably didn't need to update this, originally I thought I had gotten the osx-10-9 toolchain working so I made changes to the file with the intention of updating.

**Comment by ruslo** [comment]
> Will test this first

Looks good now, merged. Branch `pkg.mysqlclient` updated, please rebase this patch so we can test latest version.

**Comment by caseymcc** [comment]
Should be good now

**Review by ruslo — APPROVED** [review]

**Pull request merged**